core/vm/evm.go

```diff
@@ -256,6 +258,7 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 		}
 		evm.StateDB.CreateAccount(addr)
 	}
+	// we have already checked overflow before capturing snapshot, so it's safe to ignore the error here
```

Comment:
This comment is a bit confusing since the function `Transfer` does not actually return an error. Could we instead add a comment like:

```suggestion
	// It is safe to call Transfer since we have not mutated balances since calling CanTransfer
```